src/app/shared/deviceSniffingHelper.js

```diff
@@ -0,0 +1,37 @@
+const UAParser = require("ua-parser-js");
+const PHONE_TYPES = require("../../constants/phone-types");
+const OS_TYPES = require("../../constants/os-types");
+
+function modifyJourneyOnSniffing(req) {
+  const parser = new UAParser(req.headers["user-agent"]);
+
+  if (parser.getDevice()["type"] === "mobile") {
+    req.body.journey += "Smartphone";
+    switch (parser.getOS()["name"]) {
+      case OS_TYPES.IOS:
+        req.body.journey += "Iphone";
+        break;
+      case OS_TYPES.ANDROID:
+        req.body.journey += "Android";
```

Comment:
I'm a bit uncomfortable with this method modifying `req.body.journey` rather than returning a value. It feels like it's knowing too much about the request object for a device sniffing helper.

Reply:
That's fair, I'll update it to `getJourneyOnSniffing` where it returns a string